config/ini_test.go

```diff
@@ -67,7 +69,7 @@ func TestIni(t *testing.T) {
 		t.Error(pi)
 		t.Fatal(err)
 	}
-	if iniconf.String("runmode") != "dev" {
+	if iniconf.String("runmode") != beego.DEV {
```

Comment:
also I think use beego.DEV will dependence with the beego package. I just want to keep the config package independence.